src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.Document.cs

```diff
@@ -117,10 +117,17 @@ private static JsonDocument WriteDocument<TValue>(in TValue value, JsonTypeInfo<
             // For performance, share the same buffer across serialization and deserialization.
             // The PooledByteBufferWriter is cleared and returned when JsonDocument.Dispose() is called.
```

Comment:
Given the conversation we just had, I don't understand this comment.

Reply:
Once the returned `JsonDocument` is disposed by the user, the underlying buffer will be returned to the pool:

https://github.com/dotnet/runtime/blob/4f835ae4409d6c68f3af808b1d21bdb0bf037008/src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonDocument.cs#L86-L90

Reply:
Ok, then I misunderstood your previous comments.  I thought you were saying an ArrayPool array stored in JsonDocument is never returned to the pool.  You were instead saying that it's only not returned _if_ the user doesn't Dispose of the JsonDocument?  In that case, the current state of the world is fine.

Reply:
Correct. If that's acceptable I'll just close #73398.

Reply:
Yup.